README.md

```diff
@@ -26,23 +26,23 @@ across various devices by running comprehensive GitHub workflows.
 | dcgan                           | ✅    |
 | demucs                          | ✅    |
 | densenet121                     | ✅    |
-| detectron2_fasterrcnn_r_101_c4  | ❌    |
-| detectron2_fasterrcnn_r_101_dc5 | ❌    |
-| detectron2_fasterrcnn_r_101_fpn | ❌    |
-| detectron2_fasterrcnn_r_50_c4   | ❌    |
-| detectron2_fasterrcnn_r_50_dc5  | ❌    |
-| detectron2_fasterrcnn_r_50_fpn  | ❌    |
-| detectron2_fcos_r_50_fpn        | ❌    |
-| detectron2_maskrcnn             | ❌    |
-| detectron2_maskrcnn_r_101_c4    | ❌    |
-| detectron2_maskrcnn_r_101_fpn   | ❌    |
-| detectron2_maskrcnn_r_50_c4     | ❌    |
-| detectron2_maskrcnn_r_50_fpn    | ❌    |
+| detectron2_fasterrcnn_r_101_c4  | ✅    |
+| detectron2_fasterrcnn_r_101_dc5 | ✅    |
+| detectron2_fasterrcnn_r_101_fpn | ✅    |
+| detectron2_fasterrcnn_r_50_c4   | ✅    |
+| detectron2_fasterrcnn_r_50_dc5  | ✅    |
+| detectron2_fasterrcnn_r_50_fpn  | ✅    |
+| detectron2_fcos_r_50_fpn        | ✅    |
+| detectron2_maskrcnn             | ✅    |
+| detectron2_maskrcnn_r_101_c4    | ✅    |
+| detectron2_maskrcnn_r_101_fpn   | ✅    |
+| detectron2_maskrcnn_r_50_c4     | ✅    |
+| detectron2_maskrcnn_r_50_fpn    | ✅    |
```

Comment:
wow many models work

Reply:
fixed by torch-npu upgrade?

Reply:
@Yikun Not sure, as the log doesn't show the run details, just the results. Will find out more details on our npu server manually.